assets/src/edit-story/components/canvas/frameElement.js

```diff
@@ -105,8 +106,7 @@ function FrameElement({ element }) {
   return (
     <WithLink
       element={element}
-      active={selectedElementIds.length === 1 && isSelected}
-      dragging={Boolean(activeDropTargetId)}
+      active={!isSelected && hovering}
```

Comment:
Shouldn't it still appear if the element is selected + hovering? Otherwise, it appears _only_ when the element is not selected.

Reply:
It might feel weird, but it's aligned with the spec.

>2. The 2-tap ui appears on hover (not when the element is selected [Figma](https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=9466%3A5)

Reply:
Hmm, okay. It's a bit ambiguous, I read out initially from the specs that it shouldn't appear when the element is selected but should appear on hover (if the element is selected or not).

However, perhaps since in edit mode the fields are visible in the design panel, it's not necessary to show it even when hovering 🤷

Reply:
Yep, I think that's the reasoning, or it's just a hint that an element has a link, so after you select it (and you need to hover to select - so you saw the popup) - you don't need that information anymore.